projects/ltc7841/Makefile

```diff
@@ -0,0 +1,9 @@
+# Select the example you want to enable by choosing y for enabling and n for disabling
+BASIC_EXAMPLE = n
```

Comment:
Please see this pull request and use it as a template: https://github.com/analogdevicesinc/no-OS/pull/2368

The idea is that the user types `EXAMPLE=basic` or `EXAMPLE=iio_example`, these being the names of the folders where you put the example code. We will create some infrastructure to do this automatically, but for now you'll have to do it manually.